zerver/data_import/slack_message_conversion.py

```diff
@@ -347,3 +347,9 @@ def render_attachment(attachment: WildValue) -> str:
         pieces.append(f"<time:{time}>")
 
     return "\n\n".join(piece.strip() for piece in pieces if piece.strip() != "")
+
+
+def replace_links(text: str) -> str:
```

Comment:
I think this can be a separate commit too, which is done before all the bold/italic/strike-through regex changes. Just so that it's clear that this change doesn't depend on those updates.

Reply:
Split this out to : 49eca01cfc9245d3a4b2a2443a7d2179d059bbbf

Reply:
I find adding a function which isn't used yet to be a weird single commit; I'd move this change to where it's necessary (the last commit of the series)